### PR TITLE
NPM Package Registry: Added support for scoped packages

### PR DESCRIPTION
**Pull request by BlyatManGopnik**
- Added scoped package support
image

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
bump version in metadata

**Comment by BlyatManGopnik** [comment]
Did that

**Review by Bas950 — CHANGES_REQUESTED** [review]
this is not a big change v1.1.6 is enough

**Comment by BlyatManGopnik** [comment]
Fixed

**Review by Bas950 — DISMISSED** [review]

File: websites/N/NPM Package Registry/dist/metadata.json [hunk 1]
```diff
@@ -9,12 +9,12 @@
     "de": "Der Node Package Manager ist ein Paketmanager für die JavaScript-Laufzeitumgebung node.js."
   },
   "url": "www.npmjs.com",
-  "version": "1.1.5",
+  "version": "1.1.6",
   "logo": "https://i.imgur.com/XaosIBS.png",
   "thumbnail": "https://crugg.de/premid/npmbanner.png",
```
Comment (ririxidev) [inline websites/N/NPM Package Registry/dist/metadata.json:14]: could you move thumbnail to imgur? Thanks

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by BlyatManGopnik** [comment]
Done

File: websites/N/NPM Package Registry/dist/metadata.json [hunk 2]
```diff
@@ -9,12 +9,12 @@
     "de": "Der Node Package Manager ist ein Paketmanager für die JavaScript-Laufzeitumgebung node.js."
   },
   "url": "www.npmjs.com",
-  "version": "1.1.5",
+  "version": "1.1.6",
   "logo": "https://i.imgur.com/XaosIBS.png",
-  "thumbnail": "https://crugg.de/premid/npmbanner.png",
+  "thumbnail": "https://i.imgur.com/zHN2g49.png",
```
Comment (ririxidev) [inline websites/N/NPM Package Registry/dist/metadata.json:14]: could you change it to screenshot of npm site xD bc like reed isnt like promotional banner or something

**Comment by BlyatManGopnik** [comment]
done

**Review by ririxidev — APPROVED** [review]
Approved! Merry Christmas 🎉

**Review by Bas950 — APPROVED** [review]

**Pull request merged**